src/hotspot/share/gc/g1/g1GCPhaseTimes.cpp

```diff
@@ -359,11 +353,11 @@ void G1GCPhaseTimes::trace_phase(WorkerDataArray<double>* phase, bool print_sum,
 #define TIME_FORMAT "%.1lfms"
 
 void G1GCPhaseTimes::info_time(const char* name, double value) const {
-  log_info(gc, phases)("%s%s: " TIME_FORMAT, indent(1), name, value);
+  log_info(gc, phases)("%s%s: " TIME_FORMAT, "  ", name, value);
```

Comment:
I would prefer the fixed indentation spaces in the format string instead of in an extra argument. Same for the other cases.

Reply:
> I would prefer the fixed indentation spaces in the format string instead of in an extra argument. Same for the other cases.

Good.
That make sense to me.
Updated.
Thanks.